resources/eventing/charts/nats/templates/statefulset.yaml

```diff
@@ -194,6 +205,14 @@ spec:
         {{- end }}
         {{- end }}
         volumeMounts:
+        ### the secret that holds account data ###
+          {{- if and .Values.auth.enabled .Values.auth.resolver }}
+          {{- if eq .Values.auth.resolver.type "memory" }}
+          - name: resolver-volume
+            mountPath: /etc/nats-config/accounts
+          {{- end }}
+          {{- end }}
+
```

Comment:
SInce we are updating the statefulset, have you tested the upgrade flow?

Reply:
Yes, it seems to work if I upgrade from `main` to PR. The only problem so far is that IF you would actually change anything in the secret nats-server would not get updated. Usually this should not happen. A workaround is to add or remove some whitespace in the configmap which triggers the reloader.